### DISCUSSION TITLE
PulseEffects and Quod Libet

### DISCUSSION THREAD

**Discussion by LemonMunch** [quodlibet/quodlibet#3742] [Q&A] [answered]
I am trying to use PulseEffects equalizer with Quod Libet. Quod Libet does not show up as an application that is playing in PulseEffects.  Spotify and PulseEffects work fine on PC. I am on ubuntu and running alsa with this sink: alsasink device=hw:1,0.
Any ideas what could be wrong. 

Thanks

**Comment by LemonMunch** [comment]
PulseEffects and Quod Libet work together if I don't use Alsa. Is there a way to use Alsa and PulseEffects?

**Comment by declension** [comment, marked as answer]
I don't know PulseEffects. Do you mean https://github.com/wwmm/easyeffects - which seems to be a EDIT: ~PulseAudio~ PipeWire thing now?

(also note we have an gstreamer-based EQ plugin already if that's good enough)

**Reply by LemonMunch** [reply]
The EQ plugin is fine if the fixed frequencies happen to be the frequencies that need adjusting, but to EQ headphones based on the measurements (jaakkopasanen AutoEq) they have precise frequencies to adjust.  If I don't use alsa sink, PulseEffects works fine. I wanted to Alsa for bit -prefect results. I really like Quod Libet.

**Reply by LemonMunch** [reply]
Just Looked at EasyEfects looks  like a fork of PulseEffects  I will try it, 

Thanks
